conf/local.conf.sample.append

```diff
@@ -0,0 +1,30 @@
+
+#
+# meta-updater configuration, see README.adoc for more options and detailed documentation
+#
+
+MACHINE = "##MACHINE##"
+DISTRO = "poky-sota-systemd"
+
+# General SOTA setup
+#SOTA_CLIENT_PROV = "aktualizr-auto-prov"
+#SOTA_PACKED_CREDENTIALS = "/path/to/credentials.zip"
+
+# Uncomment this line to start an ssh server at boot automatically
+#IMAGE_FEATURES += "ssh-server-dropbear"
+
+# Store systemd logs in persistent storage
+#
+# It greatly helps diagnosing issues on testing devices but should be
+# carefully weighted against file system usage and flash device wear for
+# production systems. Please refer to systemd's docs for more details
+IMAGE_INSTALL_append += " systemd-journald-persistent"
+
+# Set resource limits for aktualizr service
+#
+# Uncomment these lines to change the default parameters.
+#
+#RESOURCE_CPU_WEIGHT_pn-aktualizr = "100"
+#RESOURCE_MEMORY_HIGH_pn-aktualizr = "100M"
+#RESOURCE_MEMORY_MAX_pn-aktualizr = "80%"
+IMAGE_INSTALL_append += " aktualizr-resource-control"
```

Comment:
Does it make sense to keep `aktualizr-resource-control` uncommented even if none of the variables are used?

Also, does it makes sense to have a commented-out line for adding `aktualizr-log-debug`, perhaps in the same area as the log persistence?

Reply:
The commented lines just reflect the defaults that are in aktualizr_git.bb, that's just a convenience for easily changing these values.

Agree for adding aktualizr-log-debug.